Src/Web/Web.Shared.Net/Extensibility/Implementation/PostSamplingTelemetryProcessor.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.ApplicationInsights.Web.Extensibility.Implementation
+{
+    using System.Web;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Common;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    /// <summary>
+    /// PostSamplingTelemetryProcessor evaluates deferred properties.
+    /// It is intended to be used with <see cref="RequestTrackingTelemetryModule.DisableTrackingProperties"/> after Sampling.
+    /// </summary>
+    /// <remarks>
+    /// This feature is still being evaluated and not recommended for end users.
+    /// </remarks>
+    public class PostSamplingTelemetryProcessor : ITelemetryProcessor
```

Comment:
make it internal and config will still work

Reply:
this might work for PostSamplingProcessor, however the flags will still have an issue due to [GetProperties](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/src/Microsoft.ApplicationInsights/Extensibility/Implementation/TelemetryConfigurationFactory.cs#L301). Do you suggest to change it to include BindingFlags.NonPublic ?